sections/footer.liquid

```diff
@@ -76,15 +76,16 @@
                 {%- when 'image' -%}
                   <div class="footer-block__details-content footer-block-image {{ block.settings.alignment }}">
                     {%- if block.settings.image != blank -%}
-                      {%- assign image_size = block.settings.image_width | append: 'x' -%}
+                      {%- assign image_size = block.settings.image_width -%}
```

Comment:
I wonder if we actually need `image_size` 🤔 

Maybe we could use `block.settings.image_width` instead to remove one of the assign and only use `image_size_2x` as the assign. 

Ideally we use the least amount of `assign` since `block.settings.image_width` is easy to read anyway.

Reply:
Thank you. I was hoping someone would point this out because I initially thought about doing this.

There are a couple instances of this in the footer, header, and main-article. I'll test and update.